documentation/source/user_section/tutorials/shape-analysis/normalizing-morphometrics-compressions/generating-necessary-inputs.rst

```diff
@@ -0,0 +1,84 @@
+.. _generating-necessary-inputs:
+
+Generate the necessary input files
+#############################################################
+
+The ``sct_compute_compression`` command uses the shape metric of the spinal cord at the level of compression and computes a ratio with non-compressed levels above and below the compression site.
+
+
+To use ``sct_compute_compression``, we need the 3 following input files:
+
+
+1. Spinal cord segmentation
+2. Vertebral labels
+3. Compression labels (``t2_compressed_label-compression.nii.gz``)
+
+You can also get these files by downloading :sct_tutorial_data:`data_normalizing-morphometrics-compression.zip`.
```

Comment:
Note that the current [`data_normalizing-morphometrics-compression.zip`](https://drive.google.com/file/d/1515MMcyiAqwmegpug50I5VLUrOvuU_gs/view) link is quite large, because it contains the straightening warping fields and etc. But, I think we only need a few of those files to cover the steps on this page?

I can generate a smaller `.zip` file with only the necessary files when I update the `sct_tutorial_data` dataset. :)

Reply:
yes that would be great, I included the wrong folder with everything oups!